lorawan-encoding/src/maccommands.rs

```diff
@@ -196,23 +85,27 @@ macro_rules! mac_cmds {
     ) => {
         $(
             $(#[$outer])*
-            pub struct $type<'a>(&'a [u8]);
+            pub struct $type<'a>(pub(crate) &'a [u8]);
 
             impl<'a> $type<'a> {
                 /// Creates a new instance of the mac command if there is enought data.
                 pub fn new(data: &'a [u8]) -> Result<$type<'a>, Error> {
-                    if data.len() < $size {
+                    if data.len() != $size {
```

Comment:
The error type might be misleading if the size is bigger.

Reply:
Yes, I don't know what I was thinking there :thinking: 
changing it to < would allow somebody to create the mac command from a larger buffer, leaving it as is should at least change the error message.

For now I will change it back to how it was with <